src/app/components/shell/header/header.scss

```diff
@@ -44,182 +74,583 @@
+            &::after {
+              height: 7px;
+              right: 10px;
+              margin-top: -7px;
+            }
+
+            &::after {
+              right: 6px;
+            }
+          }
 
-        a {
-            color: color(neutral-dark);
-            padding-bottom: 1rem;
-            padding-top: 1rem;
         }
 
-        .nav-menu-item {
-            flex: 0 0 auto;
-            font-size: 1.4rem;
-            font-weight: 400;
-            line-height: 0;
+        &:first-child > a {
+          @include medium-up {
+            margin-left: -2rem;
+          }
+        }
+
+        &:last-child > a {}
+
+        > a {
+          white-space: nowrap;
+          @include medium-up {
+            padding-left: 2rem;
+            padding-right: 2rem;
+
+          }
         }
+      } //end .nav-menu-item
     }
 
-    .nav {
-        align-items: center;
-        background-color: transparent;
-        color: color(neutral-medium);
+    //Navigation with Rice Logo row
+    .meta-nav {
+      order: 2;
+
+      @include medium-up {
+        order: 1;
+        background: #F0F0F0;
+        height: 50px;
+
+        @at-root .sticky#{&} {
+          position: fixed;
+          top: -70px;
+          left: 0;
+          right: 0;
+          z-index: 1;
+          height: 70px;
+          transform: translate3d(-5rem, 0px, 0);
+          background: transparent;
+        }
+      }
+
+      .container {
+        @include medium-up {
+          display: flex;
+          flex-direction: row nowrap;
+          justify-content: flex-end;
+          height: 100%;
+          align-items: center;
+        }
+      }
+      .nav-menu {
+        padding: 0 0 0 8rem;
+        margin: 0;
+        list-style: none;
+        flex: 1 100%;
+        margin: 0;
+        list-style: none;
+
+        @include small {
+          @at-root .active.open#{&} {
+            transform: translate3d(-1000px, 0, 0);
+            transition: transform 0.5s;
+          }
+
+        }
+
+        @include medium-up {
+          flex: 0 0 auto;
+          display: flex;
+          padding: 0;
+
+          @at-root .sticky.visible.active#{&} {
+            transform: translate3d(0,70px,0);
+            transition: transform 0.5s 0.6s;
+          }
+
+        }
+
+        .nav-menu-item {
+          margin: 0 0 0.5rem;
+
+          @include medium-up {
+            margin: 0;
+          }
+
+          > a {
+            color: #0DC0DC;
+            display: block;
+
+            @include medium-up {
+              color: #555759;
+              font-size: 1.4rem;
+              &:active,
+              &:focus,
+              &:hover {
+                  color: color(blue);
+              }
+            }
+
+          }
+        }
+
+
+
+      } //end nav-menu
+
+      .logo-wrapper {
+        flex: 1 100%;
+        background: transparentize(#E5E5E5, 0.8);
+        padding: 1.5rem;
+        border-top: 1px solid transparentize(#E5E5E5, 0.5);
```

Comment:
I'm not sure a `transparentize` function is necessary.  CSS's `rgba` is standard and has the exact same effect.  You can use the global variables mixed in with `rgba` as well, so you can do something like `rgba(color(gray), 0.5)`.